follow_sponsorship/view/contract_view.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+    Copyright (C) 2014 Compassion (http://www.compassion.ch)
+    @author David Coninckx <david@coninckx.com>
+    The licence is in the file __openerp__.py
+-->
+<openerp>
+	<data>
+        <record id="view_follow_contract_form_compassion" model="ir.ui.view">
+			<field name="name">follow.contract.compassion.form</field>
+			<field name="model">recurring.contract</field>
+			<field name="arch" type="xml">
+                <form string="Follow contract" version="7.0" >
+                    <header>
+                        <field name="sds_state" widget="statusbar" statusbar_visible="draft,start,waiting_welcome,active,cancelled" />

```

Comment:
Instead of cancelled being visible, put 'sub' and 'sub_accept', because we want to hope for the better scenario by default ;)